api/src/constants/email-templates/requestCompleted/html.pug

```diff
@@ -40,7 +40,12 @@ html
                             td.aligncenter
                               a.btn-primary(href=realmOMaticLink) Realm-o-Matic
                           tr
-                            td Please contact Devhub team if you have any problems or issues.
+                            td #[b If you are requesting for BCeID to be integrated], please pay attention here:
```

Comment:
If you are requesting for BCeID to be integrated, please pay attention here: BCeID identity provider will not be enabled in the production realm for you until the IDIM team has given the approval. To speed up the process, initiate a conversation with #[b IDIM team] at idim.consulting@gov.bc.ca on the details about BCeID usage ASAP!